src/components/shared/PageTitle.js

```diff
@@ -24,6 +24,15 @@ export const PageTitle = ({ title = '' }) => {
     case /\/about$/.test(path):
       pageTitle = `About`
       break
+    case /\/compendia/.test(path):
+      if (path.includes('normalized')) {
+        pageTitle = 'Normalized Compendia'
+      } else if (path.includes('rna-seq')) {
+        pageTitle = 'RNA-seq Sample Compendia'
+      } else {
+        pageTitle = 'Download Compendia -'
+      }
```

Comment:
Seems like you need to check the presence of `download` first otherwise it will always be one of the other two.